Formula/tezos-node-kathmandunet.rb

```diff
@@ -0,0 +1,87 @@
+# SPDX-FileCopyrightText: 2022 Oxhead Alpha
+# SPDX-License-Identifier: LicenseRef-MIT-OA
+
+class TezosNodeKathmandunet < Formula
+  url "file:///dev/null"
+  version "v14.0-rc1-1"
+
+  depends_on "tezos-node"
+
+  desc "Meta formula that provides background tezos-node service that runs on kathmandunet"
+
+  def install
+    startup_contents =
+      <<~EOS
+      #!/usr/bin/env bash
+
+      set -euo pipefail
+
+      node="/usr/local/bin/tezos-node"
+      # default location of the config file
+      config_file="$DATA_DIR/config.json"
+
+      mkdir -p "$DATA_DIR"
+      if [[ ! -f "$config_file" ]]; then
+          echo "Configuring the node..."
+          "$node" config init \
+                  --data-dir "$DATA_DIR" \
+                  --rpc-addr "$NODE_RPC_ADDR" \
+                  --network=kathmandunet \
```

Comment:
Same here, we should use config URL for now
```suggestion
                  --network=https://teztnets.xyz/kathmandunet \
```